salt/channel/server.py

```diff
@@ -66,6 +54,10 @@ def __init__(self, opts, transport):
         self.opts = opts
         self.transport = transport
         self.event = None
+        # self.event = salt.utils.event.get_master_event(
+        #     self.opts, self.opts["sock_dir"], listen=False
+        # )
+        # self.master_key = salt.crypt.MasterKeys(self.opts)
```

Comment:
Do you still want this commented code for future debugging?

Reply:
This is still a WIP.